cypress/e2e/util/userProfileSetup.ts

```diff
@@ -0,0 +1,75 @@
+export const setupProfile = (): Promise<any> => {
```

Comment:
We have to get the user a clean profile page login when we stub different account/org configurations for the 'only one account or only one org' use case, so I've moved the sign-in functionality to an import for DRYness.